src/app.js

```diff
@@ -48,6 +51,13 @@ const httpApp = createGatewayApp(httpServer);
 let httpsServer = createHttpsServer();
 let httpsApp = null;
 
+
+/**
```

Comment:
Comment does not match function.

Reply:
I've put a proper comment on. There are no comments on most of the functions! At least I got something! :-) :-) :-)
Done.